package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/colorpicker",
-  "version": "3.0.1",
+  "version": "3.1.0",
```

Comment:
We'll change the version in a separate pull request.

Reply:
Sounds good, I'll revert